
# Camera Parameters. Adjust them!
camera_intrinsic:
    innner: [1813.8640278523535, 0.0, 709.437703364062, 0.0, 1811.7852288202823, 497.78651196409686, 0.0, 0.0, 1.0]
    dist: [-0.06476952914945554, 0.322695509322055, -0.0014060938622962368, 0.01575836682201113, 0.0]

